imports/blocks/action-buttons/index.js

```diff
@@ -22,26 +23,28 @@ import {
   Guest as TertiaryButton,
 } from "./Buttons";
 
+const SAVED_ACCTS_QUERY = gql`
+  query GetSavedPaymentAccounts {
+    savedPayments(cache: false) {
+      name, id: entityId, date,
+      payment { accountNumber, paymentType }
+    }
+  }
+`;
 
 // XXX remove cache: false when heighliner caching is tested
-const mapQueriesToProps = ({ ownProps }) => ({
-  savedPayments: {
-    query: gql`
-        query GetSavedPaymentAccounts {
-          savedPayments(cache: false) {
-            name, id: entityId, date,
-            payment { accountNumber, paymentType }
-          }
-        }
-      `,
+const withSavedPayments = graphql(SAVED_ACCTS_QUERY, {
+  name: "savedPayments",
+  options: ownProps => ({
     variables: {
-        // even though this is unused, we include it to trigger a recal when a person
-        // logs in or logs out
+      // even though this is unused, we include it to trigger a recal when a person
+      // logs in or logs out
       authorized: ownProps.authorized,
```

Comment:
We should use the `skip: <boolean>` option here to not send the query if the user isn't authorized